### PR TITLE
fix: #6347

### PR DESCRIPTION
**Pull request by piedcipher**
fixes: #6347

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Review by sfshaza2 — COMMENTED** [review]
@domesticmouse, can you take a look?

**Comment by sfshaza2** [comment]
@piedcipher, the code is failing analysis against the stable release. Can you fix?

File: examples/cookbook/plugins/picture_using_camera/lib/main_step3.dart [hunk 1]
```diff
@@ -55,6 +56,7 @@ class TakePictureScreenState extends State<TakePictureScreen> {
     );
 
     // Next, initialize the controller. This returns a Future.
+    _initializeControllerFuture = _controller.initialize();
```
Comment (domesticmouse) [inline examples/cookbook/plugins/picture_using_camera/lib/main_step3.dart:59]: I'm curious what the intent of this future is? Will it be used by a `FutureBuilder` to update the UI once the camera controller is initialized?

**Review by domesticmouse — APPROVED** [review]

**Comment by khanhnwin** [comment]
Hi @piedcipher! Thanks for this PR! 

Can you try adding [`// ignore_for_file: unused_field`](https://dart.dev/guides/language/analysis-options#suppressing-rules-for-a-file) to the file so that the analysis ignores the `'_initializeControllerFuture' isn't used` message? That should help us get past the current build failure. Thanks!

**Comment by piedcipher** [comment]
All green! 🍏 💚

**Comment by piedcipher** [comment]
Thank you! @khanhnwin @sfshaza2

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Comment by khanhnwin** [comment]
Thanks @piedcipher! Merging.

**Pull request merged**